src/routes/Overview.svelte

```diff
@@ -21,30 +21,34 @@
 <Header>Overview</Header>
 
 <p>
-  This tool helps evaluators build Accessibility Conformance Reports in the OpenACR format.
-  It guides you through the OpenACR requirements, lets you record your software's current
-  conformance for each requirement, and generates a report.
+  This web editor helps evaluators build Accessibility Conformance Reports in the <a href="https://github.com/gsa/openacr">OpenACR format</a>.
+  It is designed to help accessibility subject matter experts create machine-readable OpenACR documents. Authors will be guided in creating an
+  accessible report for the digital product or service that they are documenting. Conformance for each requirement can be documented as required
+  to generate a Section 508 report.
+</p>
+<p>
+  <a on:click="{startNew}">Start new report</a> or open an existing one.
```

Comment:
@mgifford `startNew` is a function in the YourReport component so it doesn't work here. What is the plan for this link? The sidebar (when it is working, it is broken right now because of an error from this line of the code) will have the buttons.

Reply:
One of the observations that @danid123 & I were talking about was that the box wth the "start a new report" link ` <a on:click="{startNew}">Start new report</a> or open an existing one.` isn't always visible. The plan was to add it to the overview page, but I'll take it out now.